blog/_posts/2023-01-24-scala-center-2023-roadmap.md

```diff
@@ -0,0 +1,247 @@
+- **Make Metals work with more versions of Scala.** By creating a stable interface for the presentation compiler,
+  Metals will be less tied to the release cycles of the compiler, and it will easier to use it to develop the
+  compiler itself.
+- **Improve the Scala 3 support on Metals and IntelliJ.** We heard many developers stating that the Scala 3 support
+  in IDEs was not at the same level as Scala 2, and that is one of the blockers to migrate to Scala 3. We will
+  identify and put weigh on the main issues that currently degrade the developer experience on Scala 3 projects.
+
+Our stretch goals include: support standalone .scala files out of the box in Metals and IntelliJ (including support
+for Scala CLI configuration directives), investigate opportunities to reduce build times, reduce the maintenance costs
+of implementing the `debugSession/start` endpoint in all the build servers by running the debug server in Metals,
+update scalameta to Scala 3, publishing sbt 2, implement a Scala 2 expression evaluator in scala-debug-adapter,
+be able to choose a specific build target in Metals, make sure Scala programs are easy to deploy especially in IaaS,
+and provide TASTy trees to semantic rules in Scalafix.
+
+### Community and Contributor Experience
+
+In addition to continuing co-organizing ScalaDays, participating to conferences, publishing positive content about
+Scala, leading the Compiler Academy, and maintaining tools for contributors ([Scastie](https://scastie.scala-lang.org),
+[Scaladex](https://index.scala-lang.org), [scalafix](https://scalacenter.github.io/scalafix/)), we will:
+
+- **Add support for standalone Scala programs in Scastie.** Currently, Scastie creates an sbt project for every snippet.
+  This adds a performance overhead, and makes it impossible to just copy-paste a snippet to run it locally (because
+  you also need to re-create the sbt configuration locally). We will fix this issue by supporting a “standalone” mode
+  where the configuration will be defined via Scala CLI directives.
+- **Publish TASTy-MiMa.** This tool will be able to detect incompatibilities between two versions of a program, based
+  on their TASTy representation. We will complete its implementation and make it available to the community.
+- **Streamline the way to publish the documentation of Scala libraries.** Currently, there are many tools one can use
+  to publish documentation about their open-source projects (Scaladoc, Paradox, Laika, Docusaurus, etc.). Developers
+  should not have to study or experiment with many tools to produce the documentation they need. We will simplify how
+  to produce API documentation, how to “typecheck” the code examples, how to keep multiple versions of the documentation
+  online, and how to define custom “template directives” in Scala.
+
+Our stretch goals include: coordinate Google Summer of Code, create a Scala Center online shop, stabilize and document
```

Comment:
minor point, but I'm surprised to see GSoC considered a stretch goal

Reply:
I didn’t want to highlight it as part of our main goals because the content is already pretty dense IMHO. But we are indeed taking care of it.

Reply:
Ultimately, I’ve moved it to a regular goal here: acd54257